pkg/lwip/contrib/sys_arch.c

```diff
@@ -28,19 +28,24 @@
 #include "msg.h"
 #include "sema.h"
 #include "thread.h"
-#include "xtimer.h"
+#include "ztimer.h"
 
 #define _MSG_SUCCESS    (0x5cac)
 #define _MSG_TIMEOUT    (0x5cad)
 
+static uint32_t _elapsed_time(uint32_t start, uint32_t stop)
+{
+    return stop >= start ? stop - start : UINT32_MAX - start + stop;
```

Comment:
Won't this be fixed by integer overflow anyway?

Reply:
True!